RedHatInsights/tasks-frontend
# feat: Add quickstart buttons to upgrade and analysis tasks

Implements https://issues.redhat.com/browse/RHINENG-8305, https://issues.redhat.com/browse/RHINENG-8053, https://issues.redhat.com/browse/RHINENG-8052.

This adds "Help me get started" buttons to some tasks. The buttons open quickstarts when clicked. The buttons are hidden under feature flags so that we can wait for content to be available first.

## Screenshots

![Screenshot 2024-03-20 at 13 50 28](https://github.com/RedHatInsights/tasks-frontend/assets/31385370/75bf4cbc-c223-4054-a726-fbc7db6ddd49)

## How to test

Go to available tasks. Try to switch some of the feature flags on in the unleash stage environment and verify that "Help me get started" buttons are displayed. Clicking on them shouldn't do anything for now, because the quickstarts are not yet available.
